public/app/controllers/search-overview.controller.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+
+angular
+	.module("acpaassearch_0.0.23.controllers")
+	.controller("acpaassearchOverviewController", [
+		"$scope",
+		"$timeout",
+		"acpaassearchFactory",
+		"LabelService",
+		"NotificationService",
+
+		function(
+			$scope,
+			$timeout,
+			acpaassearchFactory,
+			LabelService,
+			NotificationService
+		) {
+			$scope.reindexing = false;
+
+			$scope.reindexSearch = function() {
+				$scope.reindexing = true;
+				acpaassearchFactory
+					.reindexSearch()
+					.then(function() {
+						NotificationService.showNotification(
+							LabelService.getString("The search is reindexing..."),
+							"top",
+							"success",
+							7000
+						);
+
+						$timeout(function() {
```

Comment:
Heb hier maar ff een timeout gezet dat ze die knop niet kunnen spammen, maar voegt in principe weinig toe, aangezien we niet weten wanneer de index klaar is :/